.github/workflows/cypress-update-snapshots.yml

```diff
@@ -0,0 +1,79 @@
+name: Cypress update snapshots
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      testname:
+        description: "Enter the test name to run (see e2e/specs/[testname].spec.js). For example: st_alert"
+        default: "st_alert"
+        type: "string"
+
+# Avoid duplicate workflows on same branch
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-cypress-update-snapshots
+  cancel-in-progress: true
```

Comment:
Since this is manually triggered & people may want to run multiple snapshot updates on the same branch simultaneously, I think we can remove the concurrency group for this workflow.
Alternatively, we could make it more specific to the spec being run:
`${{ github.workflow }}-${{ github.ref }}-${{ github.event.inputs.testname }}`

Reply:
I used the bottom solution, since this way if you accidentally run with the same `testname` twice it doesn't spawn run on a new machine.